src/Cache/InvalidCacheKeyException.php

```diff
@@ -0,0 +1,7 @@
+<?php
+namespace ScriptFUSION\Porter\Cache;
+
+class InvalidCacheKeyException extends \DomainException implements \Psr\Cache\InvalidArgumentException
```

Comment:
Unless the exception is encountered during static analysis I believe any exception is a run-time exception since it cannot be thrown without running code.